api/app/routers/sfms.py

```diff
@@ -52,7 +52,7 @@ def is_actual(filename: str) -> bool:
         year=int(file_date_string[:4]),
         month=int(file_date_string[4:6]),
         day=int(file_date_string[6:8]))
-    now = get_pst_now()
+    now = get_pdt_now()
```

Comment:
I'm confused - why the use of PDT here? Based on https://github.com/bcgov/wps/issues/1579, I thought using PDT was a big no-no?

Reply:
I know right! Timezone nightmares.
So. In this case. Here's the deal:
- If it's say, the 25th of August. Then it means we (citizens of bc) are using PDT.
- If we have a server, that makes a prediction, on the 25th of August, then we refer to that prediction as having run on the 25 of August.
- If we use a different timezone, then we may end up saying it was on a different date! So using PST, we may misattribute the date something happened on.


Such garbage. BC should just switch to PST as Glod intended.

Reply:
I'm still not able to wrap my head around this, but I don't want to have to think about it anymore. I partially trust your logic on this, and if it turns out to be wrong, well then I'll just blame you.

Reply:
Ok - switched to using "america/vancouver" - if you care to look again.